cartography/intel/aws/kms.py

```diff
@@ -321,7 +341,10 @@ def sync_kms_keys(neo4j_session, boto3_session, region, current_aws_account_id,
 
 
 @timeit
-def sync(neo4j_session, boto3_session, regions, current_aws_account_id, aws_update_tag, common_job_parameters):
+def sync(
+    neo4j_session: neo4j.Session, boto3_session: boto3.session.Session, regions: List[str], current_aws_account_id: str,
+    aws_update_tag: int, common_job_parameters: Dict,
```

Comment:
Can we standardize on `update_tag`?

Reply:
Yeah, I was gonna do that in a future PR where I homogenize variable and function names

Reply:
Roger